discord/ext/commands/bot.py

```diff
@@ -666,6 +667,8 @@ def get_prefix(self, message):
             if asyncio.iscoroutine(ret):
                 ret = yield from ret
             return ret
+        elif isinstance(prefix, collections.Iterable) and not isinstance(prefix, str):
+            return list(prefix)
```

Comment:
I would also support returning something along the lines of
```python
return list(map(str,prefix))
```
giving us the assurance that everything was cast to a string for future uses
```python
In [43]: r = 'lul'

In [44]: s = '@@'

In [45]: triggers = (r, s, '~~', 3)

In [46]: list(map(str,triggers))
Out[46]: ['lul', '@@', '~~', '3']
```